reference/test/factorization/cholesky_kernels.cpp

```diff
@@ -163,11 +188,87 @@ class Cholesky : public ::testing::Test {
                         gko::matrices::location_ani1_amd_chol_mtx);
             fn();
         }
+        {
+            SCOPED_TRACE("example");
+            this->setup(
+                {{4, 0, 1, 0, 0, 0, 0, 1, 0, 0},
+                 {0, 4, 0, 0, 1, 0, 0, 0, 0, 1},
+                 {1, 0, 4.25, 0, 0, 0, 1, 0, 0, 0},
+                 {0, 0, 0, 4, 0, 0, 0, 0, 1, 1},
+                 {0, 1, 0, 0, 4.25, 0, 0, 0, 1, 1},
+                 {0, 0, 0, 0, 0, 4, 2, 4, 0, 0},
+                 {0, 0, 1, 0, 0, 2, 5.25, 0, 0, 0},
+                 {1, 0, 0, 0, 0, 4, 0, 8, 1, 1},
+                 {0, 0, 0, 1, 1, 0, 0, 1, 4, 0},
+                 {0, 1, 0, 1, 1, 0, 0, 1, 0, 4}},
+                {{2, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+                 {0, 2, 0, 0, 0, 0, 0, 0, 0, 0},
+                 {0.5, 0, 2, 0, 0, 0, 0, 0, 0, 0},
+                 {0, 0, 0, 2, 0, 0, 0, 0, 0, 0},
+                 {0, 0.5, 0, 0, 2, 0, 0, 0, 0, 0},
+                 {0, 0, 0, 0, 0, 2, 0, 0, 0, 0},
+                 {0, 0, 0.5, 0, 0, 1, 2, 0, 0, 0},
+                 {0.5, 0, -0.125, 0, 0, 2, -0.96875, 1.67209402770897, 0, 0},
+                 {0, 0, 0, 0.5, 0.5, 0, 0, 0.598052491922453, 1.7726627476498,
+                  0},
+                 {0, 0.5, 0, 0.5, 0.375, 0, 0, 0.598052491922453,
+                  -0.448571948696326, 1.67346688755653}});
```

Comment:
I assume this solution is from matlab or other tools.
I would not like to compute it by hand.

Reply:
yes, I tried to make the solution as nice as possible, but at some point I had to give up